docs/docs/quickstarts/login/react.md

```diff
@@ -54,38 +54,40 @@ This library helps integrating ZITADEL Authentication in your React Application.
 With the installed oidc pakage you will need an AuthProvider which should contain the OIDC configuration.
 
 The oidc configuration should contain **openid**, **profile** and **email** as scope and **code** as responseType.
-In the code below the authority is already set to the issuer of zitadel.ch you can find this in the ZITADEL Console on you application.
+In the code below make sure to change the issuer to your instance url. You can find this in the ZITADEL Console on you application.
 Replace the clientId value 'YOUR-CLIENT-ID' with the generated client id of you application in ZITADEL Console.
 
-
 ```ts
-
-import React from 'react';
-import { AuthProvider } from 'oidc-react';
-import './App.css';
+import React from "react";
+import { AuthProvider } from "oidc-react";
+import "./App.css";
 const oidcConfig = {
-    onSignIn: async (response: any) => {
-        alert('You logged in :' + response.profile.given_name + ' ' + response.profile.family_name);
-        window.location.hash = '';
-    },
-    authority: 'https://issuer.zitadel.ch',
-    clientId:
-        'YOUR-CLIENT-ID',
-    responseType: 'code',
-    redirectUri: 'http://localhost:3000/',
-    scope: 'openid profile email'
+  onSignIn: async (response: any) => {
+    alert(
+      "You logged in :" +
+        response.profile.given_name +
+        " " +
+        response.profile.family_name
+    );
+    window.location.hash = "";
+  },
+  authority: "https://[your-instance].zitadel.cloud",
```

Comment:
i would make clear that this is your domain.
It dosnt need to be .zitadel.cloud
and your instance is not completly correct: its instancename-randomsting.zitadel.cloud